201-vmss-ubuntu-web-ssl/azuredeploy.json

```diff
@@ -0,0 +1,435 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmSku": {
+      "type": "string",
+      "defaultValue": "Standard_A1",
+      "metadata": {
+        "description": "Size of VMs in the VM Scale Set."
+      }
+    },
+    "ubuntuOSVersion": {
+      "type": "string",
+      "defaultValue": "14.04.4-LTS",
+      "allowedValues": [
+        "14.04.4-LTS"
+      ],
+      "metadata": {
+        "description": "The Ubuntu version for the VM. This will pick a fully patched image of this given Ubuntu version. Allowed values are: 15.10, 14.04.4-LTS."
+      }
+    },
+    "vmssName": {
+      "type": "string",
+      "metadata": {
+        "description": "String used as a base for naming resources. Must be 3-61 characters in length and globally unique across Azure. A hash is prepended to this string for some resources, and resource-specific information is appended."
+      },
+      "maxLength": 61
+    },
+    "capacity": {
+      "type": "int",
+      "metadata": {
+        "description": "Number of VM instances (100 or less)."
+      },
+      "maxValue": 100
+    },
+    "adminUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "Admin username on all VMs."
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Admin password on all VMs."
+      }
+    },
+    "vaultName": {
+      "type": "string",
+      "metadata": {
+        "description": "The Azure Key vault where SSL certificates are stored"
+      }
+    },
+    "vaultResourceGroup": {
+      "type": "string",
+      "metadata": {
+        "description": "Resource Group of the key vault"
+      }
+    },
+    "httpssecretUrlWithVersion": {
+      "type": "string",
+      "metadata": {
+        "description": "full Key Vault Id to the secret that stores the SSL cert"
+      }
+    },
+    "httpssecretCaUrlWithVersion": {
+      "type": "string",
+      "metadata": {
+        "description": "full Key Vault Id to the secret that stores the CA cert"
+      }
+    },
+    "location": {
```

Comment:
normally we recommend using resourceGroup().location instead of a location parameter (documentation here: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md), but it looks like in this case the allowed values were chosen quite specifically. If these allowed values are this specific on purpse, then this is fine as it is; if not, then please use resourceGroup().location instead :)